src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_configs.cljs

```diff
@@ -99,9 +101,12 @@
                                             (react/call :load-method-configs this)
                                             (swap! state update-in [:method-confs]
                                               assoc (:selected-index @state) new-conf)))}]
-         (:method-confs-loaded? @state) [WorkspaceMethodsConfigurationsList {:workspace (:workspace props)
-                                                                             :method-confs (:method-confs @state)
-                                                                             :parent-state state}]
+         (:method-confs-loaded? @state) [WorkspaceMethodsConfigurationsList
+                                          {:on-import
+                                             (fn [] (react/call :load-method-configs this))
```

Comment:
Make this function take the imported config as a parameter, and then if we're using mock data we can just update the list of mock configs.  Otherwise calling load-method-configs is going to blast everything.